src/components/progress-bar/__test__/index.test.js

```diff
@@ -0,0 +1,69 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { render } from '@testing-library/react';
+import '@testing-library/jest-dom';
+
+/**
+ * Internal dependencies
+ */
+import ProgressBar from './../index';
+
+describe( 'The ProgressBar component', () => {
+	it( 'renders without crashing', () => {
+		const div = document.createElement( 'div' );
+		ReactDOM.render( <ProgressBar></ProgressBar>, div );
+	} );
+
+	it( 'shows the classnames of the outer div correctly', () => {
+		const { getByTestId } = render( <ProgressBar></ProgressBar> );
+		expect( getByTestId( 'progress-bar-outer' ) ).toHaveClass(
+			'wc-free-shipping-progress-bar__outer'
+		);
+	} );
+
+	it( 'shows the classnames of the inner div correctly', () => {
+		const { getByTestId } = render( <ProgressBar></ProgressBar> );
+		expect( getByTestId( 'progress-bar-inner' ) ).toHaveClass(
+			'wc-free-shipping-progress-bar__inner'
+		);
+	} );
```

Comment:
As above, you can use `container.querySelector( '.wc-free-shipping-progress-bar__inner' ).toBeInTheDocument();

Reply:
As before, I replaced `getByTestId()` for all test cases.